### PR TITLE
BugID:1732:Add the source code of electronic scale sensor.

### PR DESCRIPTION
**Pull request by wzhiyuan-nit**
[Detail]
Issue description:
#1732 source code of electronic scale sensor
Rootcause:

Solution:

Requirement description:

Solution:

[Verified Cases]
Build Pass: <solution list>
Test Pass: <test case list>

Signed-off-by: wzhiyuan-nit <wuming.wzy@alibaba-inc.com>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : ab6cff32e27bfd95ef5701e018364a904f888d6d

**Review by ethanlcz — APPROVED** [review]
looks good to me.

**Comment by skylarCai** [comment]
auto build success,number:182

**Comment by skylarCai** [comment]
autotest fail,number:1448

File: haas_lib_bundles/python/libraries/ele_scale/README.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# 称重传感器
```
Comment (skylarCai) [inline haas_lib_bundles/python/libraries/ele_scale/README.md:1]: LGFM

File: haas_lib_bundles/python/libraries/ele_scale/ele_scale.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+from driver import GPIO
```
Comment (skylarCai) [inline haas_lib_bundles/python/libraries/ele_scale/ele_scale.py:1]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**